db/schema.rb

```diff
@@ -323,6 +323,24 @@
     t.index ["unique_id", "facility_type"], name: "index_base_facilities_on_unique_id_and_facility_type", unique: true
   end
 
+  create_table "benefits_intake_form_submissions", force: :cascade do |t|
+    t.string "form_type", null: false
+    t.uuid "benefits_intake_uuid"
+    t.uuid "submitted_claim_uuid"
+    t.jsonb "form_data", default: {}
+    t.uuid "user_account_id"
```

Comment:
if this is correctly updated to be on `saved_claims` (think it is a wip still), this should not be needed here either?

Reply:
Same as above, Simple Forms will want to keep this.